src/main/java/com/amazon/opendistroforelasticsearch/sql/esdomain/LocalClusterState.java

```diff
@@ -76,6 +83,9 @@
      */
     private final Cache<Tuple<List<String>, List<String>>, IndexMappings> cache;
 
+    /** Latest setting value for each registered key. Thread-safe is required. */
+    private final Map<String, Object> settingMap = new ConcurrentHashMap<>();
```

Comment:
I prefer not including type information in the identifier, I think it's better to call this `latestSettings`

Reply:
Done!